build/aliases.yaml

```diff
@@ -54,3 +54,6 @@ release:
   - 'shell:make_cname'
   - 'gh-pages:main'
   - 'shell:trigger_tests'
+
+check_duplicates:
+  - 'duplicate_check'
```

Comment:
Since it's only about images, better to mention that in the name. e.g. `duplicated_images`.